pkg/server/server.go

```diff
@@ -235,8 +235,8 @@ func createImageStore(db *sql.DB) store.ImageStore {
 	return store.DBImageStore{DB: db}
 }
 
-func createInstanceStore(db *sql.DB) store.InstanceStore {
-	return store.DBInstanceStore{DB: db}
+func createInstanceStore(db *sql.DB, config config.Config) store.InstanceStore {
```

Comment:
`cfg` if it's to match the rest of the file

Reply:
Good spot